### PR TITLE
ADEN-7490 Track labrador and queen to DW

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 1]
```diff
@@ -237,8 +250,9 @@ define('ext.wikia.adEngine.adContext', [
 		context.opts.netzathleten = isEnabled('wgAdDriverNetzAthletenCountries');
 		context.opts.additionalVastSize = isEnabled('wgAdDriverAdditionalVastSizeCountries');
 
-		// Need to be placed always after all lABrador wgVars checks
-		context.opts.labradorDfp = geo.mapSamplingResults(instantGlobals.wgAdDriverLABradorDfpKeyvals);
+		if (context.targeting.skin === 'oasis') {
+			trackLabrador();
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/context/adContext.js]: I'm worried if it's not too early :( (e.g. for bill-the-lizard), also if it's good place (tracking in context). What do you think about:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/run/desktop.run.js#L106

File: extensions/wikia/AdEngine/js/ml/billTheLizard.js [hunk 2]
```diff
@@ -59,6 +71,12 @@ define('ext.wikia.adEngine.ml.billTheLizard', [
 				.then(function () {
 					ready = true;
 					log(['respond'], log.levels.debug, logGroup);
+
+					var rabbitPropValue = serialize();
+
+					if (adContext.get('opts.enableAdInfoLog') && rabbitPropValue) {
+						pageInfoTracker.trackProp('rabbit', rabbitPropValue);
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/ml/billTheLizard.js]: s/rabbit/btl please

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/ml/billTheLizard.js]: Done.

File: extensions/wikia/AdEngine/js/context/adContext.js (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/context/adContext.js]: Done.

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**